src/control/cmd/dmg/pool.go

```diff
@@ -196,44 +197,63 @@ func (cmd *PoolListCmd) Execute(_ []string) (errOut error) {
 	return nil
 }
 
+type PoolID struct {
+	ui.LabelOrUUIDFlag
+}
+
 // poolCmd is the base struct for all pool commands that work with existing pools.
 type poolCmd struct {
 	logCmd
 	cfgCmd
 	ctlInvokerCmd
 	jsonOutputCmd
-	ID   string `long:"pool" required:"1" description:"Unique ID of DAOS pool"`
-	UUID string
+	uuidStr  string
+	PoolFlag PoolID `long:"pool" description:"pool UUID (deprecated; use positional arg)"`
```

Comment:
nit - probably should be "pool name or UUID"

Reply:
Yeah it's kind of a subtle hint... Even though you can use a label with `--pool`, only the positional argument help mentions label, so maybe it will steer people away from using the flag in order to get the new label feature.

Reply:
That's fine with me.